named_entity_recognition_bert/Legal-NER.ipynb

```diff
@@ -0,0 +1,1802 @@
+     "text": [
+      "              precision    recall  f1-score   support\n",
+      "\n",
+      " CASE_NUMBER       1.00      1.00      1.00       181\n",
+      "       COURT       1.00      1.00      1.00       236\n",
+      "        DATE       1.00      1.00      1.00       240\n",
+      "         GPE       1.00      1.00      1.00       216\n",
+      "       JUDGE       1.00      1.00      1.00        76\n",
+      "      LAWYER       1.00      1.00      1.00        24\n",
+      "         ORG       1.00      1.00      1.00       200\n",
+      "OTHER_PERSON       1.00      1.00      1.00       243\n",
+      "  PETITIONER       1.00      1.00      1.00        60\n",
+      "   PRECEDENT       1.00      1.00      1.00       413\n",
+      "   PROVISION       1.00      1.00      1.00       308\n",
+      "  RESPONDENT       1.00      1.00      1.00        63\n",
+      "     STATUTE       1.00      1.00      1.00       237\n",
+      "     WITNESS       1.00      1.00      1.00        83\n",
+      "\n",
+      "   micro avg       1.00      1.00      1.00      2580\n",
+      "   macro avg       1.00      1.00      1.00      2580\n",
+      "weighted avg       1.00      1.00      1.00      2580\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(classification_report(label_clean_iob, label_clean_iob))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d87781cb",
+   "metadata": {},
+   "source": [
+    "### Get predictions for unlabeled texts\n",
+    "<a id='get_predictions_for_unlabeled_texts'></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "id": "416756a9-72e1-49a6-b1a6-2e75bbb874cc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "unlabeled_texts = []\n",
+    "names = []\n",
+    "for filename in glob.glob('./unlabeled_sa_docs/*.txt'):\n",
+    "    with open(filename) as f:\n",
+    "        unlabeled_texts.append(f.read())\n",
+    "        names.append(os.path.basename(filename)) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "id": "72c87c1b-736a-4b19-a3c6-a221337bad7c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
+      "To disable this warning, you can either:\n",
+      "\t- Avoid using `tokenizers` before the fork if possible\n",
+      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
+      "To disable this warning, you can either:\n",
+      "\t- Avoid using `tokenizers` before the fork if possible\n",
+      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
```

Comment:
set the environment variable TOKENIZERS_PARALLELISM pleaseeeeeeee

Reply:
+